### PR TITLE
Update Dockerfiles and README for Chapel Release 1.20.0: Release Branch Only

### PR DESCRIPTION
**Pull request by soohoonchoi**
Updated Dockerfiles and README file to be consistent with the previous release. Updated version to 1.20.0 and kept references to current and 1 previous release.

### CODE REVIEW

**Comment by bradcray** [comment]
I'm not familiar enough with Docker to be a good reviewer here, and am about to disappear into a meeting (so don't want to be a bottleneck by trying to come up to speed).

File: util/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -21,12 +21,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
...
     && rm -rf /var/lib/apt/lists/*
 
-ENV CHPL_VERSION master
+ENV CHPL_VERSION 1.20.0
 ENV CHPL_HOME    /opt/chapel/$CHPL_VERSION
 ENV CHPL_GMP     system
 
 RUN mkdir -p /opt/chapel \
-    && wget -q -O - https://github.com/chapel-lang/chapel/archive/$CHPL_VERSION.tar.gz | tar -xzC /opt/chapel --transform 's/chapel-//' \
+    && wget -q -O - https://github.com/chapel-lang/chapel/releases/download/$CHPL_VERSION/chapel-$CHPL_VERSION.tar.gz | tar -xzC /opt/chapel --transform 's/chapel-//' \
```
Comment (ben-albrecht) [inline util/dockerfiles/Dockerfile]: What led to the URL change here?

Comment (soohoonchoi) [reply util/dockerfiles/Dockerfile]: I believe it was changed few releases ago but the master branch was not updated.  As Jenna pointed above, we'll push the change to master as well.

**Comment by bradcray** [comment]
@soohoonchoi / @ben-albrecht / @mppf: It looks to me as though the PATH variable should be updated in the Docker files to avoid warnings of the form:

```
Please update your PATH
--
chpl is now stored at /opt/chapel/1.20.0/bin/linux64-x86_64/chpl
the old path was /opt/chapel/1.20.0/bin/linux64/chpl
```

(now that we're tagging the path with the processor family).

Also, we should add Soohoon to the list of Cray contributors so that his PRs don't get tagged with the cla requirement.

**Comment by bradcray** [comment]
> It looks to me as though the PATH variable should be updated in the Docker files

Can anyone confirm or deny?  (this browser tab has been open three weeks to make sure it doesn't get forgotten...).

**Comment by mppf** [comment]
I thought it was obvious, but the chpl in the docker file should not be producing that error when you run it. I presume that updating a PATH variable in the docker file somewhere will solve it.

**Comment by bradcray** [comment]
I'd love it if someone familiar with the Docker package and/or the move of where the binary lives could fix this.

**Pull request merged**